### ISSUE TITLE
Multiplying output with every request

### ISSUE THREAD

**Issue by jpms4** [influxdata/influxdb-client-python#519]
Thanks for your great work on the influxdb-client.

I recently updated from 1.30 to 1.33 and experience a different behaviour of the client in synchronous writing. Writing to the influx database still works fine, but the request appears to be sent multiple times (or just output multiple times to stdout).

-------
Output of my script on the **fourth** request, the number of repetitions rises with every function call:

Record to be stored: [{'measurement': 'messungen', 'tags': {'source': 'measurements.xml'}, 'fields': {'AC_Voltage': 233.6, 'AC_Frequency': 50.026, 'DC_Voltage1': 1.9, 'DC_Voltage2': 4.2, 'LINK_Voltage': 1.7, 'Derating': 100.0}, 'time': '2022-10-19T18:54:31'}]

```
>>> Request: 'POST http://whisky:8086/api/v2/write?org=SchlauHaus&bucket=Solaranlagendaten&precision=ns'
>>> Request: 'POST http://whisky:8086/api/v2/write?org=SchlauHaus&bucket=Solaranlagendaten&precision=ns'
>>> Request: 'POST http://whisky:8086/api/v2/write?org=SchlauHaus&bucket=Solaranlagendaten&precision=ns'
>>> Request: 'POST http://whisky:8086/api/v2/write?org=SchlauHaus&bucket=Solaranlagendaten&precision=ns'
>>> Content-Encoding: identity
>>> Content-Encoding: identity
>>> Content-Encoding: identity
>>> Content-Encoding: identity
>>> Content-Type: text/plain
>>> Content-Type: text/plain
>>> Content-Type: text/plain
>>> Content-Type: text/plain
>>> Accept: application/json
>>> Accept: application/json
>>> Accept: application/json
>>> Accept: application/json
>>> Authorization: ***
>>> Authorization: ***
>>> Authorization: ***
>>> Authorization: ***
>>> User-Agent: influxdb-client-python/1.33.0
>>> User-Agent: influxdb-client-python/1.33.0
>>> User-Agent: influxdb-client-python/1.33.0
>>> User-Agent: influxdb-client-python/1.33.0
>>> Body: b'messungen,source=measurements.xml AC_Frequency=50.026,AC_Voltage=233.6,DC_Voltage1=1.9,DC_Voltage2=4.2,Derating=100,LINK_Voltage=1.7 1666205671000000000'
>>> Body: b'messungen,source=measurements.xml AC_Frequency=50.026,AC_Voltage=233.6,DC_Voltage1=1.9,DC_Voltage2=4.2,Derating=100,LINK_Voltage=1.7 1666205671000000000'
>>> Body: b'messungen,source=measurements.xml AC_Frequency=50.026,AC_Voltage=233.6,DC_Voltage1=1.9,DC_Voltage2=4.2,Derating=100,LINK_Voltage=1.7 1666205671000000000'
>>> Body: b'messungen,source=measurements.xml AC_Frequency=50.026,AC_Voltage=233.6,DC_Voltage1=1.9,DC_Voltage2=4.2,Derating=100,LINK_Voltage=1.7 1666205671000000000'
<<< Response: 204
<<< Response: 204
<<< Response: 204
<<< Response: 204
<<< X-Influxdb-Build: OSS
<<< X-Influxdb-Build: OSS
<<< X-Influxdb-Build: OSS
<<< X-Influxdb-Build: OSS
<<< X-Influxdb-Version: v2.2.0
<<< X-Influxdb-Version: v2.2.0
<<< X-Influxdb-Version: v2.2.0
<<< X-Influxdb-Version: v2.2.0
<<< Date: Wed, 19 Oct 2022 18:55:52 GMT
<<< Date: Wed, 19 Oct 2022 18:55:52 GMT
<<< Date: Wed, 19 Oct 2022 18:55:52 GMT
<<< Date: Wed, 19 Oct 2022 18:55:52 GMT
<<< Body: 
<<< Body: 
<<< Body: 
<<< Body: 
```

My script is roughly the following:

```
def solardaten2db(url_inv_i, url_db_i, token_db_i, org_db_i, bucket_db_i):

    # Some code to aquire the data for the record to be written to db

    client_db = influxdb_client.InfluxDBClient(url_db_i, token_db_i, org_db_i)
    write_client = client_db.write_api(write_options=SYNCHRONOUS)
    rec_db_i = [{"measurement": "messungen", "tags": {"source": "measurements.xml"}, "fields": field_listfor_db,
                 "time": timestamp_utc}]
    print("Record to be stored:",rec_db_i)
    write_client.write(bucket=bucket_db_i, org=org_db_i, record=rec_db_i)
    write_client.close()
    client_db.close()

    return error_code_i

```

**Comment by jpms4** [comment]
Thanks, urllib3 versions are 1.26.5, 1.26.10, and 1.26.12 depending on the installation I run the script on.

The sample code runs smoothly until I add "While True:" in line 4, then the output lines are repeated again with one more repetition in every loop.

**Comment by bednar** [comment]
Hi @jpms4,

I have prepared a fixed version within #521. Development version of the client will be available after #521 acceptance, stable version in regular release cycle.

You could use a fixed version by: 

`pip install git+https://github.com/influxdata/influxdb-client-python.git@fix-duplicate-logging-handler`

Regards

**Comment by jpms4** [comment]
Thanks a lot for the quick response and fix.
